paddle/trainer/TrainerConfigHelper.cpp

```diff
@@ -36,6 +36,22 @@ namespace paddle {
 
 struct TrainerConfigHelperPrivate {
   TrainerConfig conf;
+
+  /**
+   * @brief Update trainer default values to each Parameter.
+   */
+  void updateDefaultValues() {
+    // set default gradient clipping.
+    if (conf.default_values().has_gradient_clipping_threshold()) {
+      auto defaultGradientClipping =
+          conf.default_values().gradient_clipping_threshold();
+      for (auto &param : *conf.mutable_model_config()->mutable_parameters()) {
+        if (!param.has_gradient_clipping_threshold()) {
+          param.set_gradient_clipping_threshold(defaultGradientClipping);
+        }
+      }
+    }
+  }
```

Comment:
how about other parameters except `gradient_clipping_threshold`? (parameters in this issue https://github.com/PaddlePaddle/Paddle/issues/2578)